test/dummy/app/components/card/card.js

```diff
@@ -0,0 +1,3 @@
+if(typeof console !== "undefined") {
+  console.log('card')
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'console' is not defined.